support-frontend/assets/helpers/redux/checkout/product/reducer.ts

```diff
@@ -31,9 +41,6 @@ export const productSlice = createSlice({
 		setProductPrices(state, action: PayloadAction<ProductPrices>) {
 			state.productPrices = action.payload;
 		},
-		setSelectedProductPrice(state, action: PayloadAction<ProductPrice>) {
```

Comment:
After tinkering around with it for a while I decided that ultimately trying to maintain the 'selected product' in Redux state was more trouble than it's worth. A subscriptions product is essentially a _function of_ the product type, billing period, fulfilment and product options, which we can express more elegantly with a selector.